components/radio/style/index.less

```diff
@@ -13,6 +13,7 @@
 
   display: inline-block;
   line-height: unset;
+  font-size: 0;
```

Comment:
Expected "font-size" to come before "line-height" in group "Typography" (order/properties-order)